ehsm_kms_service/apis.js

```diff
@@ -0,0 +1,15 @@
+const apis = {
+  CreateKey: 'CreateKey',
+  Encrypt: 'Encrypt',
+  Decrypt: 'Decrypt',
+  GenerateDataKey: 'GenerateDataKey',
+  GenerateDataKeyWithoutPlaintext: 'GenerateDataKeyWithoutPlaintext',
+  ExportDataKey: 'ExportDataKey',
+  Sign: 'Sign',
+  Verify: 'Verify',
+  AsymmetricEncrypt: 'AsymmetricEncrypt',
+  AsymmetricDecrypt: 'AsymmetricDecrypt',
+  RA_GET_API_KEY: 'RA_GET_API_KEY',
+  RA_HANDSHAKE_MSG0: 'RA_HANDSHAKE_MSG0',
+}

```

Comment:
please separate them into two sections, e.g
const cryptographic_apis {
}

const enroll_apis {
}

Reply:
it will be fix